get-latest-tgf.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
The most portable shebang would be:
```suggestion
#!/usr/bin/env bash
```